aten/src/ATen/native/transformers/cuda/sdp_utils.cpp

```diff
@@ -248,7 +248,7 @@ bool check_flash_causal_non_square_seqlens(sdp_params const& params, bool debug)
 }
 
 
-TORCH_API bool can_use_flash_attention(sdp_params const& params, bool debug) {
```

Comment:
rebase bug

Reply:
no isnt

Reply:
I meant to remove these in previous layer of stack, TBH not sure how this doesnt' cause a bug will fix in this PR